Tests/test_util.py

```diff
@@ -1,7 +1,11 @@
+import sys
+
 from helper import unittest, PillowTestCase
 
 from PIL import _util
 
+py36 = sys.version_info.major >= 3 and sys.version_info.minor >= 6
```

Comment:
This is True for 3.6, 3.7, ... 4.6, 4.7... but False for 4.0, 4.1, ... 4.5.

How about something like sys.version_info[0:2] >= (3, 6)?

Reply:
I do like that better. Stay tuned, tweak incoming...